bmlt-versions/bmlt-versions.php

```diff
@@ -1,11 +1,11 @@
 <?php
 /*
-Plugin Name: BMLT Versions
+Plugin Name: BMLT Versions 
 Plugin URI: https://github.com/bmlt-enabled/bmlt-versions/
 Description: A simple content generator to display the versions and links of the various BMLT components. Add [bmlt_versions] to a page or a post to generate the list.
 Author: BMLT Authors
 Author URI: https://bmlt.app
-Version: 1.5.0
+Version: 1.7.0
```

Comment:
seems like we could just make this 1.6.0

Reply:
Your current version is 1.6.1
I forked before you made the change to sort by date, I just copied the php file instead of doing a pull. 
This should be 1.7.0

Reply:
Cool